test/extended/util/framework.go

```diff
@@ -318,12 +390,21 @@ func WaitForADeployment(client kclient.ReplicationControllerInterface, name stri
 		if err != nil {
 			return err, false
 		}
+		// multiple deployments are conceivable; so we look to see how the latest depoy does
+		var lastRC kapi.ReplicationController
 		for _, rc := range list.Items {
-			err, matched := okOrFailed(&rc)
-			if matched {
-				return err, false
+			// assuming won't have to deal with more than 9 deployments
+			if lastRC.GetName() <= rc.GetName() {
```

Comment:
what gets returned by getName() when lastRC is uninitialized (first time through the loop)?  i guess it returns "" and that is less than everything?